### ISSUE TITLE
[Bug]: Scheduled tasks not executing on schedule. (or at all?)

### ISSUE THREAD

**Issue by maietta** [coollabsio/coolify#2191]
### Description

I'm not sure if this is a problem between the keyboard and chair, or an actual bug.

The crontab scheduling syntax for once a minute, every minute, which is 5, space separated asterisks don't seem to be working in my case and I don't know why. My jobs have not been running for the last two days despite services being online. I've restarted and redeploy the app, with no positive change.

For now, I have to manually run the jobs via command line every so often, just to keep my Real Estate Listings on some websites up to date.

Here's screenshot showing the list of last dates the job was run and the scheduler's syntax.

![image](https://github.com/coollabsio/coolify/assets/11745020/f6eaf44e-26ef-44c2-a459-7cad3e576460)

### Minimal Reproduction (if possible, example repository)

N/A

### Exception or Error

N/A

### Version

v4.0.0-beta.277

**Comment by andrasbacsai** [comment]
I will add some fine-tuning to the scheduled jobs in the next release, but this should not happen.

If you login to your instance as the `root` user, and go to `<yourinstance>/horizon`, can you check if the `ScheduledTaskJob` are even there and failing?

**Comment by maietta** [comment]
I was able to ascertain that if I manually run the job via the command tab you've provided in the UI, the job does execute.

The 5 asterisks do denote every minute, correct, just like a regular CRON tab schedule?

Per your request:

Completed jobs are showing up here:

![image](https://github.com/coollabsio/coolify/assets/11745020/4f4bab05-b632-40b7-83b5-5ffbaee5a951)

As well as failed jobs showing up as well.

![image](https://github.com/coollabsio/coolify/assets/11745020/9bfeaa85-bca8-4fe9-a0f6-dff80ae23324)

**Comment by maietta** [comment]
Seems after the last updates, the scheduler is working on time.

**Comment by mattstein** [comment]
I hope it’s okay to follow up in this issue here!

I just got started with Coolify and I haven’t yet been able to get _any_ scheduled task to run under v4.0.0-beta.325.

![Screen Shot 2024-09-02 at 07 28 57 PM@2x](https://github.com/user-attachments/assets/d9b0a28e-9c2a-454c-85cb-497b8980979a)

Possible clues:

- Same behavior in three different resources:
    1. DocuSeal installed via service template
    2. Single-container app installed via private repository Dockerfile
    3. Multi-container app installed via public repository’s `docker-compose.yaml`
- In each case, I confirm the task works using the Execute Command tool, then create and save it to run every minute `* * * * *`, and confirm nothing happens. Then I redeploy just in case that’s required, and wait again to confirm nothing happens.
- No completed or failed `ScheduledTaskJob` items appear in Horizon at any point.
- Recent Executions always says “No executions found.” with no items.
- One of my tests would create files in a mounted volume, and I can confirm no new files are created. (Wanted to make sure it’s not secretly running and just not reporting it in the GUI.)

Anything else I can do to troubleshoot? It’s probably best to assume I did something dumb first before moving on to the hard stuff.
